src/sqlitetablemodel.h

```diff
@@ -11,6 +11,11 @@ namespace sqlb { class ForeignKeyClause; }
 class SqliteTableModel : public QAbstractTableModel
 {
     Q_OBJECT
+
+#ifdef REGEX_UNIT_TEST
+    friend class TestRegex;
```

Comment:
Well, I know what preprocessor macros are, I didn't know what that `friend` declaration does. I understood now, thanks for the great explanation!